P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -1138,20 +1136,16 @@ exemplifying here too? -->
         </closer>      
       </div>
 	  </egXML></p>
-          <p>The <gi>sex</gi> element carries a <att>value</att> attribute to give values from a project-internal taxonomy, or an
-          external standard, such as vCard's sex property <ptr target="http://microformats.org/wiki/gender-formats"/> (in which
-            <val>M</val> indicates male, <val>F</val> indicates female, <val>O</val> indicates other, <val>N</val> indicates none
-          or not applicable, <val>U</val> indicates unknown) or the often used ISO 5218:2004 <title>Representation of Human
-            Sexes</title>
-          <ptr target="http://standards.iso.org/ittf/PubliclyAvailableStandards/c036266_ISO_IEC_5218_2004(E_F).zip"/> (in which
-            <val>0</val> indicates unknown; <val>1</val> indicates male; <val>2</val> indicates female; and <val>9</val> indicates
-          not applicable, although the ISO standard is widely considered inadequate). <egXML
+          <p>The <gi>sex</gi> and <gi>gender</gi> elements carry a <att>value</att> attribute to give values from a project-internal taxonomy, or an
```

Comment:
I’m thinking `The <gi>sex</gi> and <gi>gender</gi> elements each carry a <att>value</att> attribute …`, i.e. inserting “each” so the conjunction on the L can be properly treated as singular on the R.

Reply:
Check.